packages/eslint-plugin-template/src/rules/no-call-expression.ts

```diff
@@ -35,6 +51,15 @@ export default createESLintRule<Options, MessageIds>({
 
         if (isChildOfBoundEvent) return;
 
+        if (
```

Comment:
Would it make sense to create a helper function for this logic, like seen in other rules? Something like `functionNameIsAllowed` could make this easier to understand. What do you think?

Reply:
👍 good call, would keep this concise. I will update.